From 1394c16f3617697e059bd1405bb358611554e317 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 6 Mar 2018 09:37:11 +0100
Subject: [PATCH] basic/cgroup-util: remove unused variable

(cherry picked from commit eef03d70c1bedb3aabd4e2bcf10ab1f2678443bf)
---
 src/basic/cgroup-util.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/basic/cgroup-util.c b/src/basic/cgroup-util.c
index 9a4dd72270..c0962f288f 100644
--- a/src/basic/cgroup-util.c
+++ b/src/basic/cgroup-util.c
@@ -2038,7 +2038,6 @@ int cg_get_keyed_attribute(
                 char **ret_values) {
 
         _cleanup_free_ char *filename = NULL, *contents = NULL;
-        _cleanup_fclose_ FILE *f = NULL;
         const char *p;
         size_t n, i, n_done = 0;
         char **v;
-- 
2.17.0

